packages/cfpb-forms/src/atoms/text-input.less

```diff
@@ -1,3 +1,79 @@
+/* Border and  Outline Status styling */
+.border-outline-error {
```

Comment:
For helper mixins like this we sometimes prefix them with `u-` to indicate that it's a one-off utility thing. I think that would help with these border helpers you have here. See a similar example usage in https://github.com/cfpb/design-system/blob/main/packages/cfpb-typography/src/atoms/links.less#L66